tests/plugins/keyword_test.py

```diff
@@ -227,105 +218,24 @@ def test_analyze_objective_c_positives(self, file_content):
-        ],
-    )
-    def test_analyze_javascript_negatives(self, file_content):
-        assert not KeywordDetector().analyze_line(filename='mock_filename.js', line=file_content)
-
-    @pytest.mark.skip(
-        reason='TODO: false positive heuristics need to be migrated over to filters/*',
-    )
-    @pytest.mark.parametrize(
-        'file_content',
-        STANDARD_NEGATIVES + [
-            # FOLLOWED_BY_EQUAL_SIGNS_RE
-            '$password = $input;',
-        ],
-    )
-    def test_analyze_php_negatives(self, file_content):
-        assert not KeywordDetector().analyze_line(filename='mock_filename.php', line=file_content)
-
-    @pytest.mark.skip(
-        reason='TODO: false positive heuristics need to be migrated over to filters/*',
-    )
-    @pytest.mark.parametrize(
-        'file_content, file_extension',
-        (
-            (negative, file_extension)
-            for negative in (
-                STANDARD_NEGATIVES + [
-                    # FOLLOWED_BY_COLON_QUOTES_REQUIRED_RE
-                    'apikey: hope]nobody[finds>-_$#thisone',
-                    'apikey:hope]nobody[finds>-_$#thisone',
-                    'theapikey:hope]nobody[finds>-_$#thisone',
-                    # FOLLOWED_BY_EQUAL_SIGNS_QUOTES_REQUIRED_RE
-                    "some_dict['secret'] = hope]nobody[finds>-_$#thisone",
-                    'my_password=hope]nobody[finds>-_$#thisone',
-                    'my_password= hope]nobody[finds>-_$#thisone',
-                    'my_password =hope]nobody[finds>-_$#thisone',
-                    'my_password = hope]nobody[finds>-_$#thisone',
-                    'my_password =hope]nobody[finds>-_$#thisone',
-                    'the_password=hope]nobody[finds>-_$#thisone\n',
-                ]
-            ) for file_extension in QUOTES_REQUIRED_FILE_EXTENSIONS
         ),
     )
-    def test_analyze_quotes_required_negatives(self, file_content, file_extension):
-        assert not KeywordDetector().analyze_line(
-            filename=f'mock_filename{file_extension}',
-            line=file_content,
-        )
+    def test_analyze_javascript_negatives(line):
+        with transient_settings({
+            'plugins_used': [{
+                'name': 'KeywordDetector',
+            }],
+        }):
+            assert not list(scan_line(line))
 
-    @pytest.mark.skip(
-        reason='TODO: false positive heuristics need to be migrated over to filters/*',
-    )
-    @pytest.mark.parametrize(
-        'file_content, file_extension',
-        (
-            (standard_positive, file_extension)
-            for standard_positive in STANDARD_POSITIVES
-            for file_extension in (
-                '.yaml',
-                '.yml',
-            )
-        ),
-    )
-    def test_analyze_yaml_negatives(self, file_content, file_extension):
-        assert not KeywordDetector().analyze_line(
-            filename=f'mock_filename{file_extension}',
-            line=file_content,
-        )
```

Comment:
@KevinHock : do you remember why this test was supposed to behave as such? I spent some time reading the code, but I couldn't understand why YAML files would be treated differently, and that lines that would have been caught in a non-YAML file will be ignored with YAML files.

Reply:
I added {{ to prefix the standard positives test, and think I made this test to test we skipped them.

I think this had to do specifically with 
```
        ) or (
            filetype == FileType.YAML
            and lowered_secret.startswith('{{')
            and lowered_secret.endswith('}}')
        )
```
at the time

https://github.com/Yelp/detect-secrets/commit/592d077084f4f66d4175d2e6b7055a8529f0ade7#diff-e6de82b29d51ff0d50bb9a4951479046f43f6dad3d40ba27e3dd7e2a9f6fda8bR226

Reply:
Yeah, that's so bizarre.

Looks like it was [removed](https://github.com/Yelp/detect-secrets/commit/a53d12e65c450f5daea44290e7ba0cee744fe686#diff-e6de82b29d51ff0d50bb9a4951479046f43f6dad3d40ba27e3dd7e2a9f6fda8bL308-L310) though, so I don't know how tests kept passing.

I'm going to leave it out for now, and migrate the filters based on the master branch, and the active false positive filters on this plugin.